### DISCUSSION TITLE
Can't define Environment protection rules

### DISCUSSION THREAD

**Discussion by bmetea** [community/community#37252] [Actions]
My company is on the Team plan.

According to this [document](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment), we should be able to define environment reviewers and a wait timer but none of these options is available to me.

![image](https://user-images.githubusercontent.com/3688690/197860260-9a869ba4-8123-4c12-8287-1755ac6176a0.png)

At the same time, whenever I run my workflow, during setup, secret source is actions rather than environment, is this an indication that it's not picking the secrets from the right place ?
![image](https://user-images.githubusercontent.com/3688690/197860893-1165ea7e-eb14-42de-bf5c-fc122e498707.png)

**Comment by devminded** [comment]
It says in the document you linked to that Environment Protection rules for private repos is only available in GitHub Enterprise.

> For access to other **_environment protection rules_** in **_private or internal repositories_**, you must use GitHub Enterprise.
